### PR TITLE
Feature (Docs): Adding documentation for Docker Service Kill Experiment

### PR DESCRIPTION
**Pull request by ghoshankur1983**
Signed-off-by: Ankur Ghosh (ankur.ghosh3@wipro.com)

Adding the Documentation for Docker Service Kill experiment.

### CODE REVIEW

File: docs/docker-service-kill.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+    <td> Generic </td>
+    <td> Kills the docker service on the application node to check the resiliency. </td>
+    <td> GKE </td>
+  </tr>
+</table>
+
+## Prerequisites
+
+- Ensure that the Litmus Chaos Operator is running by executing `kubectl get pods` in operator namespace (typically, `litmus`). If not, install from [here](https://docs.litmuschaos.io/docs/getstarted/#install-litmus)
+- Ensure that the `docker-service-kill` experiment resource is available in the cluster  by executing                         `kubectl get chaosexperiments` in the desired namespace. If not, install from [here](https://hub.litmuschaos.io/api/chaos/1.5.0?file=charts/generic/docker-service-kill/experiment.yaml)
```
Comment (ispeakc0de) [inline docs/docker-service-kill.md]: can we check for the hub link? it should be 1.6.0

**Review by ispeakc0de — CHANGES_REQUESTED** [review]

**Comment by ghoshankur1983** [comment]
Added the change. Please close the PR if found suitable.

**Review by uditgaurav — DISMISSED** [review]
lgtm

**Comment by ispeakc0de** [comment]
@ghoshankur1983 Charts PR has been merged. Can you please embedded the chaos-engine & RBAC for this doc. Please run `embedmd -w docs/docker-service-kill.md`  --- to install the embedmd use `go get github.com/campoy/embedmd`

**Comment by ghoshankur1983** [comment]
I had performed the steps. However I do not know what to expect post running the steps. The steps completed without any output in the console. Can I verify anything to ensure the successful completion?

**Comment by ispeakc0de** [comment]
@ghoshankur1983 Yes, the expected behavior is same as you are getting for RBAC. It will embed the corresponding manifest here. But I am unable to see a few sections like: `Sample ChaosEngine Manifest`, `Supported Experiment Tunables` and `Prepare ChaosEngine`

File: docs/docker-service-kill.md [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+
+### Create the ChaosEngine Resource
+
+- Create the ChaosEngine manifest prepared in the previous step to trigger the Chaos.
+
+  `kubectl apply -f chaosengine.yml`
+
+- If the chaos experiment is not executed, refer to the [troubleshooting](https://docs.litmuschaos.io/docs/faq-troubleshooting/) 
+  section to identify the root cause and fix the issues.
+
```
Comment (uditgaurav) [inline docs/docker-service-kill.md:109]: There should be one chaosengine link here as well like this:
`[embedmd]:# (https://raw.githubusercontent.com/litmuschaos/chaos-charts/master/charts/generic/docker-service-kill/engine.yaml yaml)`
and after doing embedmd it should as show yaml snipt of engine.yaml

**Review by uditgaurav — APPROVED** [review]

**Review by ispeakc0de — APPROVED** [review]

**Review by ksatchit — APPROVED** [review]

**Pull request merged**